MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,565 @@
+
+{% endtabs %}
+
+## Change toolbar item visibility
+
+Hide or show the toolbar items by specifying their icon names and setting the boolean values to true or false.
+
+N> Customize an icon by specifying its `Name`.
+
+{% tabs %}
+
+{% highlight C# %}
+
+        SfImageEditor imageEditor = new SfImageEditor();
+        imageEditor.Source = ImageSource.FromFile("image.png");
+        imageEditor.SetToolbarItemVisibility("Shape", false);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Change toolbar item enable/disable state
+
+Enable or disable the toolbar items by using the `IsEnabled` property in the `ImageEditorToolbarItem` class.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+
+ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
+ImageEditorToolbarGroupItem browseGroup = (ImageEditorToolbarGroupItem) headerToolbar.ToolbarItems[0];
+ImageEditorToolbarItem browseItem = browseGroup.Items.FirstOrDefault();
+browseItem.IsEnabled = false;
+
+ImageEditorToolbarGroupItem saveGroup = (ImageEditorToolbarGroupItem) headerToolbar.ToolbarItems[2];
+ImageEditorToolbarItem saveItem = saveGroup.Items.FirstOrDefault(i => i.Name == "Save");
+saveItem.IsEnabled = false;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Customize toolbar item view
+
+To customize the view of toolbar items, specify their icon names and set the view using the `View` property in the `ImageEditorToolbarItem`.
+
+{% tabs %}
+
+{% highlight C# %}
+
+Image image = new Image();
+image.Source = ImageSource.FromResource("MyProject.Resources.Images.save.jpg");
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+
+ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
+ImageEditorToolbarGroupItem saveGroup = (ImageEditorToolbarGroupItem)headerToolbar.ToolbarItems[1];
+ImageEditorToolbarItem saveItem = zoomGroup.Items.FirstOrDefault(i => i.Name == "Save");
+saveItem.View = image;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add SubToolbar
+
+Create `SubToolbars` within your toolbar items to organize related actions.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image4.png");
+ImageEditorToolbar footerToolbar = imageEditor.Toolbars[1];
+ImageEditorToolbarItem cropItem = (ImageEditorToolbarItem)footerToolbar.ToolbarItems.FirstOrDefault(i => i.Name == "Crop");
+cropItem.SubToolbarOverlay = false;
+cropItem.SubToolbars = new List<ImageEditorToolbar>()
+{
+    new ImageEditorToolbar()
+    {
+        ToolbarItems = new List<IImageEditorToolbarItem>()
+        {
+           new ImageEditorToolbarItem(){ Name = "Rectangle"},
+           new ImageEditorToolbarItem(){ Name = "Pen"},
+           new ImageEditorToolbarItem(){ Name = "AddText"}
+        },
+    }
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add an item with the existing toolbar.
```

Comment:
## Add an Item with the Existing Toolbar.

Reply:
Updated